invenio_pages/assets/semantic-ui/js/invenio_pages/search/SearchResultItem.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2023 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { Table } from "semantic-ui-react";
+import isEmpty from "lodash/isEmpty";
+import { Actions } from "@js/invenio_administration";
+import { withState } from "react-searchkit";
+import { AdminUIRoutes } from "@js/invenio_administration";
+import { Formatter } from "@js/invenio_administration";
+import _truncate from "lodash/truncate";
+
+class SearchResultItemComponent extends Component {
+  refreshAfterAction = () => {
+    const { updateQueryState, currentQueryState } = this.props;
+    updateQueryState(currentQueryState);
+  };
+
+  render() {
+    const {
+      title,
+      resourceName,
+      result,
+      columns,
+      displayEdit,
+      displayDelete,
+      actions,
+      idKeyPath,
+      resourceSchema,
+      listUIEndpoint,
+    } = this.props;
+    const resourceHasActions =
+      displayEdit || displayDelete || !isEmpty(actions);
+    result.content = _truncate(result.content, { length: 250 });
+
+    return (
+      <Table.Row>
+        {columns.map(([property, { text, order }], index) => {
+          return (
+            <Table.Cell
+              key={`${text}-${order}`}
+              data-label={text}
+              className="word-break-all"
+            >
+              {index === 0 && (
+                <a
+                  href={AdminUIRoutes.detailsView(
+                    listUIEndpoint,
+                    result,
+                    idKeyPath
+                  )}
+                >
+                  <Formatter
```

Comment:
Is this formatter needed here?

I see there are three [formatters](https://github.dev/inveniosoftware/invenio-administration/blob/e3a8cbf68260a9f0ef68776375a119c197a34a92/invenio_administration/assets/semantic-ui/js/invenio_administration/src/components/Formatter.js#L14) (`date`, `datetime` and `bool`) but `pages` uses the `url` as the first column.

Reply:
minor: to avoid code duplication consider something like [element variables](https://reactjs.org/docs/conditional-rendering.html#element-variables):

```javascript
const formattedElement = <Formatter result={result} resourceSchema={resourceSchema} property={property}/>;
const columnElement = index > 0 ? formattedElement : <a href="...">{formattedElement}<a/>;

return (
   {columnElement}
)
```

However you might only "shrinken" the `Formatter` element and not the conditional part. 

Imo both options save some space, given that ~50% of the component is a conditional + the `Formatter` element whose instantiation is duplicated.